cmd/tailscaled/tailscaled.service

```diff
@@ -20,9 +20,15 @@ CacheDirectory=tailscale
 CacheDirectoryMode=0750
 Type=notify
 
+BindPaths=/dev/net/
+DeviceAllow=/dev/net/tun
+DeviceAllow=/dev/null
```

Comment:
Also need /dev/random and /dev/urandom, unless those are implicitly allowed by something else.

Reply:
Right. I only saw these two in my strace log, but I guess {u,}random is used when setting up Tailscale to generate certs and such? And yeah, `DevicePolicy=closed` includes {u,}random and null, but `DevicePolicy=strict`, as I understand it, exclude even these. Either way, if they’re used at some point, they should be allowed.